careers/ladder.md

```diff
@@ -563,6 +578,65 @@ See
 [this document](https://www.notion.so/artsy/Artsy-Tech-Lead-Expanded-Responsibilities-Public-710ee0cd2fce47dcaa722ece80e365b2)
 for an expanded set of Tech Lead Responsibilities at Artsy.
 
+## Staff Engineer Archetypes
+
+### Optimizers / Fixers / Solver
+
+These engineers love to dive deep into a complex problem and resurface with a solution that is accessible to team
+mates.
+
+### Product Eengineers
+
+These engineers are passionate about delivering value to users, find crafty ways to overcome roadblocks to shipping
+product, and are very comfortable collaborating with non-technical stakeholders.
```

Comment:
To add: Something about how this archetype feels comfortable helping to shape the product strategy based on their knowledge of technical and user needs.

A toy example could be: Identifying a new capability being rolled out to iOS, seeing how it would fit into Artsy's near-term business and product strategy, and working with PMs to plan and prioritize the work amongst product teams.